packages/mask/src/plugins/FileService/Worker/swarm.ts

```diff
@@ -0,0 +1,92 @@
+import { Attachment } from '@dimensiondev/common-protocols'
+import { encodeText } from '@dimensiondev/kit'
+import { Bee, CollectionEntry } from '@ethersphere/bee-js'
+import urlcat from 'urlcat'
+import { isEmpty } from 'lodash-unified'
+import { landing } from '../constants'
+import type { ProviderAgent, LandingPageMetadata, AttachmentOptions } from '../types'
+import { makeFileKeySigned } from '../helpers'
+
+const POSTAGE_STAMP = '0000000000000000000000000000000000000000000000000000000000000000'
+const BEE_HOSTS: string[] = [
+    'https://bee-0.gateway.ethswarm.org',
+    'https://bee-1.gateway.ethswarm.org',
+    'https://bee-2.gateway.ethswarm.org',
+    'https://bee-3.gateway.ethswarm.org',
+    'https://bee-4.gateway.ethswarm.org',
+    'https://bee-5.gateway.ethswarm.org',
+    'https://bee-6.gateway.ethswarm.org',
+    'https://bee-7.gateway.ethswarm.org',
+    'https://bee-8.gateway.ethswarm.org',
+    'https://bee-9.gateway.ethswarm.org',
+]
+
+function createBee(): Bee {
+    const randomIndex = Math.floor(Math.random() * BEE_HOSTS.length)
+    return new Bee(BEE_HOSTS[randomIndex])
```

Comment:
```suggestion
    return new Bee(sample(BEE_HOSTS))
```

see https://lodash.com/docs/4.17.15#sample